fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/response/DruidResultRow.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+/**
+ * A row of results in a Druid Response.
+ */
+public abstract class DruidResultRow {
+    @JsonIgnore
+    private final String timestamp;
```

Comment:
Having searched Fili codebase, I see timestamp is modeled using `DateTime`, instead of String, everywhere such as `Result`, `TopNResultSetMapper`, `DataSourceMetadataLoader`.

Having it as `DateTime` is also better so that child class can select different timezone instead of fixing to `DateTimeZone.UTC`